content/master/software/install.md

```diff
@@ -269,6 +269,7 @@ at the table below.
 | Alpha | `--enable-composition-webhook-schema-validation` | Enable Composition validation using schemas. |
 | Alpha | `--enable-environment-configs` | Enable support for EnvironmentConfigs. |
 | Alpha | `--enable-external-secret-stores` | Enable support for External Secret Stores. |
+| Alpha | `--enable-usages` | Enable support for Usages. |
```

Comment:
@plumbis This one-liner causes the Vale errors below: 

<img width="1243" alt="Screenshot 2023-08-17 at 17 56 39" src="https://github.com/crossplane/docs/assets/9900707/89ea6c08-c05a-42e9-9b57-3ca705db2c4b">

Am I expected to resolve all of those?